extensions/xcode/src/components/xcode-projects-menu-bar/xcode-projects-menu-bar.component.tsx

```diff
@@ -0,0 +1,19 @@
+import { MenuBarExtra } from "@raycast/api";
+import { usePromise } from "@raycast/utils";
+import { XcodeProjectsMenuBarItem } from "./xcode-projects-menu-bar-item.component";
+import { XcodeProjectMenuBarService } from "../../services/xcode-project-menu-bar.service";
+
+/**
+ * Xcode Projects Menu Bar
+ */
+export function XcodeProjectsMenuBar(): JSX.Element {
+  const { isLoading, data } = usePromise(XcodeProjectMenuBarService.xcodeProjects);
+  return (
+    <MenuBarExtra isLoading={isLoading} icon="xcode-menu-bar-icon.png" tooltip="Show Recent Xcode Projects">
+      <MenuBarExtra.Item title="Recent Xcode Projects" />
```

Comment:
very small suggestion - feel free to ignore though :D 
```suggestion
      <MenuBarExtra.Item title="Recent Xcode Projects" />
      <MenuBarExtra.Separator />
```